CMakeLists.txt

```diff
@@ -480,6 +481,10 @@ if(${IREE_BUILD_EMBEDDING_SAMPLES} AND NOT ${IREE_BUILD_SAMPLES})
   add_subdirectory(iree/samples/simple_embedding)
 endif()
 
+if(${IREE_BUILD_STATIC_LIBRARY_SAMPLES} AND NOT ${IREE_BUILD_SAMPLES})
+  add_subdirectory(iree/samples/static_library)
```

Comment:
move if()'s like this to the cmake file itself - we don't want a 10000 line top level cmake. 
you'd move this to the top of your cmakelists:
```
if(NOT "${IREE_BUILD_STATIC_LIBRARY_SAMPLES}" AND NOT "${IREE_BUILD_SAMPLES}")
  return()
endif()
```
(or whatever we settle on)

Reply:
Moved up to the stati_library cmakelists.txt for now.